posthog-core/src/index.ts

```diff
@@ -144,6 +147,7 @@ export abstract class PostHogCoreStateless {
     // Init promise allows the derived class to block calls until it is ready
     this._initPromise = Promise.resolve()
     this._isInitialized = true
+    this.disableCompression = !isGzipSupported() || (options?.disableCompression ?? false)
```

Comment:
`lz64` is always supported and its a compression, so most likely we should not check for `isGzipSupported` here

Reply:
nvm, `lz64` is deprecated and will be removed at some point